cpp/include/Ice/Initialize.h

```diff
@@ -290,6 +290,15 @@ namespace Ice
          */
         std::function<void()> threadStop;
 
+#ifdef __APPLE__
+        /**
+         * Whether or not to use a dispatch queue for ThreadPool execution. If true, each {@link ThreadPool} will
```

Comment:
We should mention the platforms this is supported, when looking at the API reference it would not be clear that this is APPLE only. And same for other public methods that are platform-specific.